fields/class-cmb-hidden-field.php

```diff
@@ -0,0 +1,46 @@
+<?php
+/**
+ * Hidden field type.
+ *
+ * @package WordPress
+ * @subpackage Custom Meta Boxes
+ */
+
+/**
+ * Hidden field type.
+ *
+ * @since 1.1.0
+ *
+ * @extends CMB_Field
+ */
+class CMB_Hidden_Field extends CMB_Field {
+
+	/**
+	 * Print out a field wrapper.
+	 *
+	 * Overriding this as there's no reason nor opportunity to have a repeatable hidden field and we don't
+	 * want to display the title and description here so getting rid of all of that logic.
+	 */
+	public function display() {
+		?>
+
+		<div class="field-item" data-class="<?php echo esc_attr( get_class( $this ) ); ?>" style="display: none; <?php echo esc_attr( $this->args['style'] ); ?>">
+
+			<?php $this->html(); ?>
+
+		</div>
+
+		<?php
+	}
```

Comment:
Yes, I'd like to keep the markup consistent and it's used for JS checks so if we needed to make any transformations to the field in the future it keeps it consistent.